.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,16 @@
+---
+<!--
+Remove the fields that are not appropriate
+Please include:
+-->
+
+**What's this PR do?**
+
+**Any background context you want to provide (if appropriate)?**
+
+**Screenshots (if appropriate)?**
```

Comment:
@cPhost we can remove (if appropriate here) since we have already included that in header.

Reply:
Done.